### ISSUE TITLE
com.hazelcast.jet.core.ClusterStateChangeTest.when_enterPassiveState_then_jobTerminated

### ISSUE THREAD

**Issue by cangencer** [hazelcast/hazelcast-jet#1199]
https://hazelcast-l337.ci.cloudbees.com/view/Jet/job/jet/com.hazelcast.jet$hazelcast-jet-core/989/testReport/junit/com.hazelcast.jet.core/ClusterStateChangeTest/when_enterPassiveState_then_jobTerminated/

```
Error Message
Job status expected:<NOT_RUNNING> but was:<STARTING>
Stacktrace
java.lang.AssertionError: Job status expected:<NOT_RUNNING> but was:<STARTING>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at com.hazelcast.jet.core.ClusterStateChangeTest.when_enterPassiveState_then_jobTerminated(ClusterStateChangeTest.java:86)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:98)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by viliam-durina** [comment]
Will be fixed by #1181

**Comment by viliam-durina** [comment]
Fixed by #1181
